app/components/Form/ProjectEmissionIntensityReportForm.tsx

```diff
@@ -24,6 +24,46 @@ interface Props {
   onSubmit: () => void;
 }
 
+// passing content suffix to fields that need nominator and denominator (example: tCO2e/GJ)
+const createEmissionIntensityReportUiSchema = (
+  emissionFunctionalUnit: string,
+  productionFunctionalUnit?: string
+) => {
+  // Example: tCO2e/GJ if we have emissionFunctionalUnit
+  const contentSuffix: JSX.Element = emissionFunctionalUnit && (
+    <b>{`${emissionFunctionalUnit}${
+      productionFunctionalUnit ? `/${productionFunctionalUnit}` : ""
+    }`}</b>
+  );
+  return {
+    ...emissionIntensityReportUiSchema,
+    baselineEmissionIntensity: {
+      ...emissionIntensityReportUiSchema.baselineEmissionIntensity,
+      "ui:options": {
+        contentSuffix,
+      },
+    },
+    targetEmissionIntensity: {
+      ...emissionIntensityReportUiSchema.targetEmissionIntensity,
+      "ui:options": {
+        contentSuffix,
+      },
+    },
+    postProjectEmissionIntensity: {
+      ...emissionIntensityReportUiSchema.postProjectEmissionIntensity,
+      "ui:options": {
+        contentSuffix,
+      },
+    },
+    totalLifetimeEmissionReduction: {
+      ...emissionIntensityReportUiSchema.totalLifetimeEmissionReduction,
+      "ui:options": {
+        contentSuffix,
```

Comment:
We can confirm with the team, but this one should always be just `tCO2e` , not the rate.

Reply:
Is this related to the comment below?!
https://github.com/bcgov/cas-cif/pull/915#discussion_r936113669

Reply:
It was 👍